flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1182,6 +1206,7 @@ def teardown_example(self, token):
         """
         Cleanup after running a hypothesis example.
         """
+        umount_all(self.deployer.mountroot)
```

Comment:
Nice catch! That's actually a pre-existing bug... umount_all is already used in contexts where it is expected to be synchronous (people call addCleanup on SyncronousTestCasts with umount_all... I'll look into how to fix that.